script/main.js

```diff
@@ -305,17 +305,17 @@ function getInitialState() {
     device = 'mobile';
   }
 
-  // Set defaults as median.
   const metricValues = {};
-  const metricScorings = {...scoringGuides.v6.desktop, ...scoringGuides.v5.desktop};
+  // If no metric values come in w/ params, initalize with mobile medians (score of 50)
+  const metricScorings = {...scoringGuides.v5.mobile, ...scoringGuides.v6.mobile}; // v5 is neccessary for FCI
   for (const id in metricScorings) {
     metricValues[id] = metricScorings[id].median;
   }
 
-  // Load from query string.
+  // Populate metricValues from query string.
   for (const [id, metric] of Object.entries(metrics)) {
     const value = params.get(id) || params.get(metric.auditId);
-    if (value === undefined) continue;
+    if (value === null) continue;
```

Comment:
```suggestion
    if (!value) continue;
```

?

Reply:
but... empty strings and 0s.

`URLSearchParams.get()` always returns null if it's not present.

Reply:
oh right.

well then isn't `params.get(id) || params.get(metric.auditId);` wrong for the same reason? really we'd want `??` but not supported.

Reply:
> well then isn't `params.get(id) || params.get(metric.auditId);` wrong for the same reason?

yes true. good catch

Reply:
wait, no, because it isn't 0, it's '0'.

And empty string seems like something we'd want to ignore

Reply:
yeah i realized that it's '0', too. 

so.. anyway with this PR as merged... you can have a url like `#FCP=2290&SI=2850&FMP=4000&TTI=&FCI=6500&LCP=2080&TBT=260&CLS=0.12&device=mobile&version=6&version=5`  

see that `TTI=` (empty string)..  

in this case, the TTI value is populated with the default, whereas the rest are not. i don't think this is ideal, but ... i also don't think this situation is likely to happen so i kinda don't care.

tldr: whatevz